### PR TITLE
Redesign

### PR DESCRIPTION
**Pull request by rbth7e5**

### CODE REVIEW

File: public/src/pages/Home.js [hunk 1]
```diff
@@ -2,46 +2,46 @@ import React from "react";
...
-      );
-    }
+    welcome = `Welcome ${user.username}.`;
+    description = "You are registered as";
+    user.type
+      .filter((t) => t !== "caretaker")
+      .forEach((t, i) => {
+        description += ` ${
+          i === user.type.length - 1 && user.type.length !== 1 ? "and" : ""
+        } a ${USER_TYPES[t]}${i === user.type.length - 1 ? "." : ","}`;
```
Comment (weikiat97) [inline public/src/pages/Home.js:36]: wat dis do ah i catch no ball

Comment (weikiat97) [reply public/src/pages/Home.js:36]: but ok if it works sure

Comment (rbth7e5) [reply public/src/pages/Home.js:36]: print the description lor. Like the roles. Need to make sure the last user type ends with a full stop and starts with "and"

Comment (weikiat97) [reply public/src/pages/Home.js:36]: ok nice gonna merge dis

**Pull request merged**